src/Applications/beta/examples/Remove-MgBetaApplicationFederatedIdentityCredential.md

```diff
@@ -2,17 +2,7 @@
 
 ```powershell
Import-Module Microsoft.Graph.Beta.Applications
 
-$params = @{
-	name = "testing02"
-	issuer = "https://login.microsoftonline.com/3d1e2be9-a10a-4a0c-8380-7ce190f98ed9/v2.0"
-	subject = "a7d388c3-5e3f-4959-ac7d-786b3383006a"
-	description = "Updated description"
-	audiences = @(
-		"api://AzureADTokenExchange"
-	)
-}
-
-Update-MgBetaApplicationFederatedIdentityCredential -ApplicationId $applicationId -FederatedIdentityCredentialId $federatedIdentityCredentialId -BodyParameter $params
+Get-MgBetaApplicationFederatedIdentityCredential -ApplicationId $applicationId -FederatedIdentityCredentialId $federatedIdentityCredentialId

```

Comment:
Cmdlet mismatch - `Remove-MgBetaApplicationFederatedIdentityCredential.md` vs `Get-MgBetaApplicationFederatedIdentityCredential`

Reply:
Thanks for noticing this issue. I am currently reviewing the logic